src/content/docs/alerts-applied-intelligence/applied-intelligence/anomaly-detection/custom-anomalies.mdx

```diff
@@ -0,0 +1,104 @@
+---
+title: Custom anomaly detection
+tags:
+  - Alerts and applied intelligence
+  - Applied intelligence
+  - Proactive detection
+metaDescription: Learn how custom anomaly detection in New Relic notifies you of unusual app behavior. 
+---
+
+import alertsHowCustomAnomaliesWork from 'images/alerts_screenshot-full_how-custom-anomalies-work.png'
+
+import alertsAnomaliesSetUpperandLowerRanges from 'images/alerts_screenshot-full_anomalies-set-upper-and-lower-ranges.png'
+
+import alertsFacetedAnomaliesTwo from 'images/alerts_screenshot-full_faceted-anomalies-two.png'
+
+import alertsFacetedAnomalies from 'images/alerts_screenshot-full_faceted-anomalies.png'
+
+import alertsConfigureYourAnomalyThresholds from 'images/alerts_screenshot-full_configure-your-anomaly-thresholds.png'
+
+import iconAlertCritical from 'images/icon-alert-critical.png'
+
+import iconAlertWarning from 'images/icon-alert-warning.png'
+
+Custom anomalies allow your team the most flexibility when detecting unusual behavior in your system. Not only are they flexible and dynamic, custom anomalies provide your team with the capability to alert on any NQRL condition and to adjust and optimize your thresholds. Custom anomalies are built using the same advanced tuning settings as static alerting so you can ensure your team sees only incidents that are important to you. 
+
+To create a custom anomaly, first you must create a condition. Go to **[one.newrelic.com](https://one.newrelic.com/all-capabilities) > Alerts & AI > Policies > (create or select policy) > Create alert condition**: You can select APM, Browser, or NRQL condition. 
```

Comment:
Not sure why the colon is here at the end of the file path? Maybe you can add "to create a condition" at either the beginning or the end of that sentence?

Reply:
Nevermind on this comment, I think just removing the colon is all you need. I forgot you did what I suggested in the sentence before.

Reply:
Done.